themes/cypress/languages/en.yml

```diff
@@ -19,63 +19,65 @@ page:
 
 sidebar:
   guides:
-    overview: Overview
-    why-cypress: Why Cypress?
-    getting-started: Getting Started
-    key-differences: Key Differences
-    installing-cypress: Installing Cypress
-    writing-your-first-test: Writing Your First Test
-    testing-your-app: Testing Your App
-    interacting-with-elements: Interacting with Elements
-    core-concepts: Core Concepts
-    introduction-to-cypress: Introduction to Cypress
-    writing-and-organizing-tests: Writing and Organizing Tests
-    conditional-testing: Conditional Testing
-    test-runner: The Test Runner
-    dashboard: Dashboard
-    dashboard-introduction: Introduction
-    projects: Projects
-    runs: Runs
-    organizations: Organizations
-    users: Users
-    github-integration: GitHub Integration
     analytics: Analytics
-    debugging: Debugging
-    variables-and-aliases: Variables and Aliases
-    stubs-spies-and-clocks: Stubs, Spies, and Clocks
-    network-requests: Network Requests
-    screenshots-and-videos: Screenshots and Videos
-    guides: Guides
+    assertions: Assertions
+    best-practices: Best Practices
+    bundled-tools: Bundled Tools
+    changelog: Changelog
+    code-coverage: Code Coverage
     command-line: Command Line
+    conditional-testing: Conditional Testing
+    configuration: Configuration
     continuous-integration: Continuous Integration
-    parallelization: Parallelization
-    environment-variables: Environment Variables
-    launching-browsers: Launching Browsers
+    core-concepts: Core Concepts
     cross-browser-testing: Cross Browser Testing
     custom-commands: Custom Commands
-    web-security: Web Security
-    references: References
-    configuration: Configuration
-    experiments: Experiments
-    proxy-configuration: Proxy Configuration
+    dashboard-introduction: Introduction
+    dashboard: Dashboard
+    debugging: Debugging
+    environment-variables: Environment Variables
     error-messages: Error Messages
-    bundled-tools: Bundled Tools
-    assertions: Assertions
-    best-practices: Best Practices
-    trade-offs: Trade-offs
-    troubleshooting: Troubleshooting
-    changelog: Changelog
-    migration-guide: Migration Guide
-    roadmap: Roadmap
-    tooling: Tooling
+    experiments: Experiments
+    getting-started: Getting Started
+    github-integration: GitHub Integration
+    guides: Guides
     IDE-integration: IDE Integration
+    installing-cypress: Installing Cypress
+    interacting-with-elements: Interacting with Elements
+    introduction-to-cypress: Introduction to Cypress
+    key-differences: Key Differences
+    launching-browsers: Launching Browsers
+    migration-guide: Migration Guide
+    migration: Migration
+    module-api: Module API
+    network-requests: Network Requests
+    organizations: Organizations
+    overview: Overview
+    parallelization: Parallelization
     plugins-guide: Plugins
+    projects: Projects
+    protractor-migration-guide: Protractor Migration Guide
```

Comment:
You should probably add the new definition to the other language files. `ru.yml`, `zh-cn.yml`, etc. It will work without it, but...it'll be missing when someone is translating the titles.

Reply:
Weird. It is added locally... not sure why it's not showing up in the PR